images/rekognition.liquid.html

```diff
@@ -0,0 +1,135 @@
+<script src="https://assets.crowd.aws/crowd-html-elements.js"></script>
```

Comment:
Can we please rename the template names to be more specific. For example rekognition_detect_moderation.liquid.html instead of rekoginition.liquid.html and same applies for Textract template as well